bundles/WebToPrintBundle/CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing to Pimcore Web To Print Bundle
+As an open source project we love to work together with our community to improve and develop our products. 
+It's also important for us to make clear that **we're not working for you or your company**, 
+but we enjoy to work together to solve issues or improve the product. 
+
+## Contributing Pull Requests
```

Comment:
please keep it simple like this [one](https://github.com/pimcore/customer-data-framework/blob/4.x/CONTRIBUTING.md) as we don't have mentioned branches on the repo